src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -94,19 +104,27 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     // Get the input and time from the form.
     String textEntryString = request.getParameter("text");
-    String moodScaleString = request.getParameter("mood");
     String songEntryString = request.getParameter("song");
     String artistEntryString = request.getParameter("artist"); 
     long timestamp = System.currentTimeMillis();
     String email = currentUser.getEmail();
 
     // Ensure that form is filled out before saving to datastore
     if (textEntryString != null && !textEntryString.isEmpty()) {
-      // Convert the mood input to an int.
-      int moodScale = Integer.parseInt(moodScaleString);
+      // Get Sentiment Analysis of Journal Entry
+      int moodScale = 0;
+      try{
+        moodScale = analyzeSentimentText(textEntryString);
+      }catch (Exception e) {
```

Comment:
Nit: add space after try and before catch

Reply:
Done :)